### DISCUSSION TITLE
PURCHASE CANCELLATION AND REFUND REQUEST

### DISCUSSION THREAD

**Discussion by Marcos4211234** [aclap-dev/video-downloadhelper#317] [License & Purchase requests]
### What's your issue?

I CANNOT DOWNLOAD THE VIDEOS I WANT. I REQUEST A REFUND OF MY PURCHASE, I MADE IT LESS THAN 30 MINUTES.

### The 10 first digits (only 10) of your license number or email address

0a19c1f837

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
